lib/api_valve/forwarder/response.rb

```diff
@@ -9,9 +9,10 @@ module ApiValve
   # to the original caller
 
   class Forwarder::Response
+    # FIXME: does response use permissions or check against them (here or in gem clients)?
```

Comment:
The permissions are available in the response, so they can be used to remove data from the response (if it is desired).

Reply:
OK, then I'll remove this comment :)